app/models/region.rb

```diff
@@ -53,6 +53,52 @@ def protected_areas_per_governance
     }
   end
 
+  def protected_areas_per_iucn_category
+    region_data = Hash.new { |hash, key| hash[key] = Hash.new }
+    processed_data = []
+    total_region_count = []
+
+    countries.each do |country|
+      country.protected_areas_per_iucn_category.each do |protected_area|
+        region_pa_category = region_data[protected_area["iucn_category_name"]]
+        region_pa_category["count"] ||= 0
+        region_pa_category["count"] += protected_area["count"].to_i
+        total_region_count << protected_area["count"].to_i
+        region_pa_category["percentage"] ||= 0
+        region_pa_category["percentage"] += protected_area["percentage"].to_f
+      end
+    end
+
+    processed_data = region_data.map{ |key,value| {
+          "iucn_category_name" => key,
+          "count" => value["count"],
+          "percentage" => 100 * value["count"] / total_region_count.reduce(0, :+)
```

Comment:
It's preferred to use symbols for the hashes over strings where possible https://stackoverflow.com/questions/8189416/why-use-symbols-as-hash-keys-in-ruby

Reply:
Thanks @spencerldixon the only thing is that if we choose to use symbols we need to rewrite the other parts of the code which are already using strings. I went this path so we could unify the partials. If you want we can rewrite the other code which was already there, but we need to do that if we choose to use symbols in the same shared partial.